src/NUnitEngine/nunit.engine.tests/Drivers/NUnit3FrameworkDriverTests.cs

```diff
@@ -51,21 +51,6 @@ public void CreateDriver()
             _driver = new NUnit3FrameworkDriver(AppDomain.CurrentDomain, assemblyName);
         }
 
-        //[Test]
-        //public void ConstructController()
-        //{
-        //    Assert.That(_controller..Builder, Is.TypeOf<DefaultTestAssemblyBuilder>());
-        //    Assert.That(_controller.Runner, Is.TypeOf<DefaultTestAssemblyRunner>());
-        //    Assert.That(_controller.AssemblyPath, Is.EqualTo(MOCK_ASSEMBLY));
-        //    Assert.That(_controller.Settings, Is.SameAs(_settings));
-        //}
-
-        public void ConstructController_MissingFile_ThrowsArgumentInvalid()
-        {
-            var assemblyName = typeof(NUnit.Framework.TestAttribute).Assembly.GetName();
-            Assert.That(new NUnit3FrameworkDriver(AppDomain.CurrentDomain, assemblyName), Throws.ArgumentException);
-        }
-
```

Comment:
This is unrelated, but this test was missing its TestAttribute so not running. The test is no longer valid.